### PR TITLE
Cookbook react structure

### PR DESCRIPTION
**Pull request by pabloferro**
#### New cookbook repo structure

```
|- api
|- app <- cookbook app
  |- src
  |- ...
  |- package.json
|- cookbook-<tech>  <- one recipe folder per tech
  |- recipes
```

I went for one folder per tech so every tech could have a working project to develop their components.

#### Pending
* Get techs by query (currently they are harcoded in the select).
* Allow to browse components of every tech at the same time (currently, you must filter by tech).
* Allow techs to customize its recipe folder via `.cookbook.json`?
* If we are going to keep the "web" tech, I think we should restore the pre commit scss processing.

## Screenshots
techs

## Trello Card
https://trello.com/c/exqDxyqv/49-reorganizaci%C3%B3n-del-proyecto-a-monorepo

### CODE REVIEW

File: app/src/constants/interfaces/recipe.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+}
+
+// TODO: Recipe should have one of readme, config, or html as required
+export interface Recipe {
+  title: string;
+  html: RecipeFile;
+  css: RecipeFile;
+  scss: RecipeFile;
+  readme?: RecipeFile;
+  config: {
```
Comment (mcallegari10) [inline app/src/constants/interfaces/recipe.ts:25]: We should create a type for the config? (BTW I think we can call it `cookbook.json`? I prettey like that 😝)

File: cookbook-react/.woloxci/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,27 @@
+FROM node:8.5.0
```
Comment (mcallegari10) [inline cookbook-react/.woloxci/Dockerfile:1]: I think we don't want this `.woloxci` folder for the cookbook. Just with the husky should be enough? What do you think?

File: cookbook-react/README.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
+# Components
+
+This branch contains all the components the Cookbook App will be rendering and displaying.
```
Comment (mcallegari10) [inline cookbook-react/README.md:3]:
```suggestion
This branch contains all the React components the Cookbook App will be rendering and displaying.
```

Comment (mcallegari10) [reply cookbook-react/README.md:3]: Oh, we'll need to change this README. Add a `TODO` please

File: cookbook-react/.woloxci/Dockerfile (hunk 2 above)
Comment (pabloferro) [reply cookbook-react/.woloxci/Dockerfile:1]: If it works I'd keep it to be honest.

File: cookbook-react/README.md (hunk 3 above)
Comment (pabloferro) [reply cookbook-react/README.md:3]: I've created a card for this and the other pending things to do.

**Pull request merged**